project/api/models.py

```diff
@@ -1,27 +1,52 @@
 from django.core.exceptions import ValidationError
 from django.contrib.gis.db import models
 from simple_history.models import HistoricalRecords
+from colorfield.fields import ColorField
 
 # Create your models here.
 class Nation(models.Model):
-    name = models.SlugField(max_length=20)
-    local_name = models.CharField(max_length=30)
-    wikipedia = models.URLField()
+    """
+    Cultural/governmental entity. Serves as foreign key for most Territories
+    """
+    name = models.CharField(max_length=20,
+                            help_text="Canonical name, should not include any epithets, must be unique",
```

Comment:
This name will not be visible to the users, right? Probably in the URL on the frontend

Reply:
I was planning on having it be the display name

Reply:
Hmm how about e.g. "Sao Tome and Principe" which is more than 20 chars?

I'd make this field a slug, and the display name could be the 1st occurence in aliases, what do you think?

Reply:
Also, just thinking out loud: maybe a slug field would also be useful?

On the frontend, I'm imagining urls to be like `/{nation_slug}/1970/1/1` or something similar. `nation_id` would of course work too, but looks way less readable.

Reply:
Something to consider is that in Postgres, VARCHAR always better than or equal to CHAR (https://dba.stackexchange.com/questions/126003/index-performance-for-char-vs-varchar-postgres).

That is why I only used `TextField`, never `CharField` in my initial models (https://github.com/interactivemap/interactive-map-backend/blob/master/mappy/models.py).

Reply:
Good idea, yeah 20 is pretty short in retrospect lol. 100 it is.

@dwaxe Interesting link. The only reason I was using Char was because it renders in the Admin panel as a textbox rather than a textarea